src/activities/components/ActivityItem.vue

```diff
@@ -206,6 +206,13 @@ export default {
       leaveDialog: false,
     }
   },
+  computed: {
+    host: function () {
+      // added to the ICS url so that it still works in Cordova
+      // see https://github.com/yunity/karrot-frontend/issues/2400
+      return window.location.origin
```

Comment:
Maybe directly return the complete url here?
```js
return `${window.location.origin}/api/activities/${this.activity.id}/ics/`
```

Then the computed property `host` could be renamed to `icsUrl` and used in the template...
```js
<QBtn
  :href="icsUrl"
>
```